src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertSuppliers.sql

```diff
@@ -68,28 +68,28 @@ BEGIN
 
     INSERT INTO [dbo].[SupplierContact] ([Id], [SupplierId],[FirstName],[LastName],[Email],[PhoneNumber],[LastUpdated],[LastUpdatedBy])
     VALUES
```

Comment:
While you making changes to this file could you remove the square brackets and make sure there's a space after each comma in the `INSERT` statement?

Reply:
Updated format of Insert statement